library/src/G4CMPKaplanQP.cc

```diff
@@ -433,6 +440,21 @@ G4CMPKaplanQP::CalcQPAbsorption(G4double qpE,
   return EDep;
 }
 
+
+// Handle quasiparticle energy-dependent absorption efficiency
+
+G4double
+G4CMPKaplanQP::CalcQPEfficiency(G4double qpE) const{
```

Comment:
Space before the opening brace.  I also think you can fit this whole declaration on one line.

Reply:
Also, of course this function needs to be declared in the .hh file.